udata_hydra/db/resource.py

```diff
@@ -1,9 +1,24 @@
+from typing import Optional
+
 from udata_hydra import context
 
 
 class Resource:
     """Represents a resource in the "catalog" DB table"""
 
+    STATUSES = {
```

Comment:
I think adding a `STALE` with a verbose description (as in not currently being checked) could make it clearer than leaving it to None?

Reply:
```suggestion
    STATUSES = {
        "CRAWLING_URL": "resource URL currently being crawled",
        "CRAWLED_URL": "resource URL crawl finished",
    }
```
I'm not sure why some tasks have a "finished" status while others don't

Reply:
@maudetes added a `None: "no status, waiting"` status

Reply:
@Pierlou the only status which had a "finished" status was CRAWLED, on purpose, because several stuff could happen between CRAWLED and TO_PROCESS_RESOURCE, but actually after reading more I was confused with the crawling step. So I removed it. Thanks!